pkgs/development/python-modules/sympy/default.nix

```
@@ -8,11 +8,11 @@
 
 buildPythonPackage rec {
   pname = "sympy";
-  version = "1.6.2";
+  version = "1.7.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1cfadcc80506e4b793f5b088558ca1fcbeaec24cd6fc86f1fdccaa3ee1d48708";
+    sha256 = "sha256-o96SYel1Nbg7uGB7DaLH0DEmZQ+v6isniWV7Ipwkay4=";
```

If the SRI hash format is not allowed, you can use this instead:
```suggestion
    sha256 = "0bkb4jf24yv5i4kjpsmg1xjjccfhqyi0syv0p0xvhdbmx5hr5pm3";
```
(converted by `nix to-base32 sha256-o96SYel1Nbg7uGB7DaLH0DEmZQ+v6isniWV7Ipwkay4=`)

---

There's a lot of packages with SRI hash in nixpkgs, so I believe it's allowed.

---

OK. Thank you for confirmation.